### PR TITLE
Add a changelog for 0.2.0

### PR DESCRIPTION
**Pull request by gnufied**
Add a changelog for 0.2.0

### CODE REVIEW

File: CHANGELOG-0.2.0.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# v0.2.0
```
Comment (msau42) [inline CHANGELOG-0.2.0.md]: To follow Kubernetes style, "Changelog since v0.1.0"

File: CHANGELOG-0.2.0.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+
+# Changelog since v0.1.0
```
Comment (msau42) [inline CHANGELOG-0.2.0.md]: This should go after support status, and then everything nested one more level

Comment (gnufied) [reply CHANGELOG-0.2.0.md]: fixed

**Pull request merged**